addons/sale/data/sale_data.xml

```diff
@@ -6,19 +6,5 @@
             <field name="use_invoices" eval="True"/>
             <field name="dashboard_graph_model">sale.report</field>
         </record>
-
-        <record id="sale_layout_cat_1" model="sale.layout_category">
```

Comment:
1 commit (as first commit) to remove the feature would have been better. It makes the review and the code understanding much easier. 

With a merge commit at the end. Do you think that you could do it ?

Reply:
It's a good idea and I have no doubt I can do it, the only question is how much time do I get.

Reply:
We decided with AL it wasn't worth the time.